Sources/DeepLearning/Loss.swift

```diff
@@ -54,18 +54,32 @@ func _vjpSoftmaxCrossEntropy<Scalar: TensorFlowFloatingPoint>(
 /// - Parameters:
 ///   - logits: One-hot encoded outputs from a neural network.
 ///   - oneHotLabels: One-hot encoded values that correspond to the correct output.
-@differentiable(wrt: logits, vjp: _vjpSoftmaxCrossEntropy)
+@available(*, deprecated, renamed: "softmaxCrossEntropy(logits:probabilities:)")
```

Comment:
Should we just delete this? We provide no backwards-compatibility guarantees right now, and I think we should aggressively delete stuff to keep things clean.

Reply:
Sure, works for me.

Reply:
Deleted.